modules/installation-registry-storage-block-recreate-rollout.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_vsphere/installing-vsphere.adoc
+// * installing/installing_vsphere/installing-vsphere-installer-provisioned-customizations.adoc
+// * installing/installing_vsphere/installing-vsphere-installer-provisioned-network-customizations.adoc
+// * installing/installing_vsphere/installing-vsphere-installer-provisioned.adoc
+// * installing/installing_vsphere/installing-vsphere-network-customizations.adoc
+// * installing/installing_vsphere/installing-vsphere.adoc
+// * installing/installing_vsphere/installing-restricted-networks-vsphere.adoc
+// * registry/configuring_registry_storage/configuring-registry-storage-vsphere.adoc
+
+[id="installation-registry-storage-block-recreate-rollout_{context}"]
+= Configuring block registry storage for VMware vSphere
+
+To allow the image registry to use block storage types such as AWS Elastic Block Store (EBS) during upgrades as a cluster administrator, you can use the `Recreate` rollout strategy.
+
+.Procedure
+
+. To set the image registry storage as a block storage type, patch the registry so that it uses the `Recreate` rollout strategy and runs with only `1` replica:
++
+----
+$ oc patch config.imageregistry.operator.openshift.io/cluster --type=merge -p '{"spec":{"rolloutStrategy":"Recreate","replicas":1}}'
+----
++
+. Provision the PV for the block storage device, and create a PVC for that  volume. The requested block volume uses the ReadWriteOnce (RWO) access mode.
++
+. Edit the registry configuration so that it references the correct PVC, as described in the instructions for xref:../../registry/configuring_registry_storage/configuring-registry-storage-vsphere.adoc#configuring-registry-storage-vsphere[Configuring the registry for vSphere].
```

Comment:
It's tedious, but should this go in the assembly versus the module (for now)?

Reply:
Hmm, that's a good question, @bmcelvee. I did a small rewrite of this other preceding module to introduce this added block-provisioning procedure: https://github.com/openshift/openshift-docs/pull/24043/files#diff-590288643623e120e7f40f6999df4ae5R42-R44

I'm inclined to leave `modules/installation-registry-storage-block-recreate-rollout.adoc` as a module because it can stand on its own, but I'm open to either way. Would you place it at the end of the assembly, just before `Additional resources`?

Reply:
This is definitely better as a module - you're right. For these types of links, I've been putting the sentence with the link in the assembly immediately following the include. So when it builds it looks like it is part of the module. An `Additional resources` section seems clunky for situations like these. This workaround is also probably bending the xref rules about as far as I can. :)

Reply:
Thanks for the clarification, @bmcelvee! Makes sense now..._pounds it in head_...no xrefs in modules. :)

I made some small modifications based on your suggestion. Couldn't figure out a way to indent the added note so that it appears inline below #3. But I think it should work. LMK if this looks good.

Reply:
LGTM! Thank you! :)